contracts/tests.cairo

```diff
@@ -0,0 +1,3 @@
+mod test_asserts;
```

Comment:
is this file necessary?

Reply:
At the moment yes, because their pipeline expects to find a file called tests.cairo that will then itself call the test files.
Quite annoying, I agree.